doc/appendices/command-line/traffic_layout.en.rst

```diff
@@ -102,6 +102,10 @@ Below is an example of customized yaml file (custom.yml) to construct. ::
 
 If ``traffic_layout init --layout="custom.yml"`` is executed, a runroot following the format above will be created.
 
+.. Note::
+
+   Please update the directory for cache in ``storage.config`` according to the customized runroot.
```

Comment:
":file:`storage.config` does not use the ``cachedir`` value, but makes its relatives paths relative to the base prefix".